network-api/networkapi/wagtailpages/templates/wagtailpages/index_page.html

```diff
@@ -57,6 +57,11 @@ <h3 class="tw-body result-info">
 
     <div class="index-entries row">
     {% else %}
+
+    {% if not filtered %}
+      {% block entry_list_title %}{% endblock%}
+    {% endif %}
+
```

Comment:
**Suggestion** I think we should avoid introducing special cases in parent templates that are only needed for one specific child. You can extended the content of a block by using something like this in the child template:

```html+django
{% block theblockname %}
     <h2>The block did not have a heading before</h2>
    {{ block.super }}
{% endblock theblockname %}
```

See also: https://docs.djangoproject.com/en/4.0/ref/templates/language/#template-inheritance

Reply:
I did not know this and appreciate that you sent me the documentation around this, I gave it a look and found it very useful. Thanks!